apps/web/src/pages/templates/components/StepNameInput.tsx

```diff
@@ -17,6 +17,7 @@ export const StepNameInput = ({ index, defaultValue }: { index: number; defaultV
     <Controller
       control={control}
       name={`steps.${index}.name`}
+      defaultValue=""
```

Comment:
missed default value, it's required by the `react-hook-form`... there are few more fields like this